src/components/Search.vue

```diff
@@ -132,4 +127,28 @@ export default {
     }
   }
 }
+
+.girder-arrow-menu {
```

Comment:
How can we refactor this class for use elsewhere in Girder Web Components?

I created MenuActivatorBtn because I wanted to use this type of popover style elsewhere.  This isn't tightly coupled to search, so I think it might be worth investigating how to have some global style.

If we decide that this style isn't abstract enough to refactor globally, could we rename it `.girder-search-arrow-menu`.  I personally think it can be refactored.

Reply:
Renaming is a good idea.
This girder-arrow-menu is not far from being able to be used globally, but it is still a missing option to be aligned left. I think when there is another use case for the arrow button, we could revisit and consider making it a global utility then.